.github/npm-publish-release.yml

```diff
@@ -0,0 +1,22 @@
+name: npm-publish
+on:
+  release:
+    types: [published]
+jobs:
+  publish:
+    environment: 'Publish to NPM'
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      id-token: write
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: '22'
```

Comment:
Is Node.js 22 enough? Don’t we need one version before and one version after?

Reply:
not sure what you mean here - this is just the latest version avaialble via github actions to run the npm commands. we could change/downgrade, but it's something that would need to be adjusted when that version is EOL

Reply:
current default is v20: https://github.com/actions/setup-node/blob/main/action.yml

i suppose we could also use that `check-latest` key and not worry about changing the version in the future

Reply:
I had this in mind: https://github.com/stacks-network/rendezvous/blob/8747c6276c8c693a955405e33b7c5642756da6b8/.github/workflows/ci.yml#L18

Now I realize that’s something else so you’re correct! Thanks! 🙏